src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1035,15 +1035,33 @@ public Response toggleSuperuser(@PathParam("identifier") String identifier) {
 		ActionLogRecord alr = new ActionLogRecord(ActionLogRecord.ActionType.Admin, "toggleSuperuser")
 				.setInfo(identifier);
```

Comment:
Another note at the top. Can you please document this near https://preview.guides.gdcc.io/en/develop/api/native-api.html#make-user-a-superuser ?

Reply:
@pdurbin done